assembly/src/release/bin/activemq

```diff
@@ -109,14 +96,7 @@ if [ -z "$ACTIVEMQ_BASE" ] ; then
   ACTIVEMQ_BASE="$ACTIVEMQ_HOME"
 fi
 
-# Configure user specified classpath here or externally using this variable
-if [ -z "$ACTIVEMQ_USER_CLASSPATH" ] ; then
-    ACTIVEMQ_USER_CLASSPATH=""
-fi
-
-# ActiveMQ Classpath configuration
-ACTIVEMQ_CLASSPATH="$ACTIVEMQ_BASE/../lib/:$ACTIVEMQ_USER_CLASSPATH"
-
+ACTIVEMQ_CLASSPATH="$ACTIVEMQ_BASE/../lib/"
```

Comment:
The user should be able to define its classpath extend, no ? What's the rationale here ?

Reply:
No, that doesn't seem to be necessary from my point of view.